cloudsmith_cli/cli/commands/auth.py

```diff
@@ -58,3 +69,47 @@ def authenticate(ctx, opts, owner):
             debug=opts.debug,
         )
         auth_server.handle_request()
+
+        if not token:
+            return
+
+        initialise_api()
+        try:
+            api_token = user.create_user_token_saml()
+            click.echo(
+                f"New token value: {click.style(api_token['key'], fg='magenta')}"
+            )
+            create, has_errors = create_config_files(
+                ctx, opts, api_key=api_token["key"]
+            )
+            new_config_messaging(has_errors, opts, create, api_key=api_token["key"])
+            return
+        except exceptions.ApiException as exc:
+            if exc.status == 400:
+                click.confirm(
+                    "User already has a token. Would you like to recreate it?",
```

Comment:
Suggest rewording this msg to make it clear the old token will stop working (at least, until multiple-token support is implemented)